### PR TITLE
Add firework boost for elytras!

### LINKED ISSUE #197: Elytra firework boost not working
**Describe the bug**
Firework cant be used to boost Elytra flight speed

**To Reproduce**
1. Glide with an Elytra
2. Right click with Firework
3. Firework is used but no speed is gained

**Expected behavior**
When a firework is used while flying with an Elytra, a speed boost is expected.

**Server version**
Paper version git-Paper-126

**Geyser version**
Geyser inventory # 7

**Bedrock version**
v1.14.30

### PR DESCRIPTION
**Pull request by OnlyBMan**
Adds firework boosts! 🎆🎆🎆 
Fixes #197

### CODE REVIEW

**Comment by RednedEpic** [comment]
🎆 🎆 🎆

**Comment by OnlyBMan** [comment]
Oops forgot to say, fixes #197

File: connector/src/main/java/org/geysermc/connector/entity/FireworkEntity.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+public class FireworkEntity extends Entity {
+
+    public FireworkEntity(long entityId, long geyserId, EntityType entityType, Vector3f position, Vector3f motion, Vector3f rotation) {
+        super(entityId, geyserId, entityType, position, motion, rotation);
+    }
+
+    @Override
+    public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession session) {
+        if (session.getInventory().getItem(session.getInventory().getHeldItemSlot() + 36).getId() == ItemTranslator.FIREWORK_ROCKET && session.getPlayerEntity().getMetadata().getFlags().getFlag(EntityFlag.GLIDING)) {
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/entity/FireworkEntity.java:45]: According to https://wiki.vg/Entity_metadata#Fireworks There is an optional entity ID in metadata ID 8 that is used when gliding. Can you use that to test if a player is gliding?

**Comment by Chew** [comment]
> Oops forgot to say, fixes #197

You need to edit the original message for it to actual connect it to the issue

File: connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java [hunk 2]
```diff
@@ -49,6 +49,9 @@
     // Shield ID, used in Entity.java
     public static final int SHIELD = 829;
 
+    //Firework ID, used in FireworkEntity.java
+    public static final int FIREWORK_ROCKET = 778;
+
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java:54]: These are unused, so can be removed

File: connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java [hunk 3]
```diff
@@ -49,6 +49,9 @@
     // Shield ID, used in Entity.java
     public static final int SHIELD = 829;
 
+    //Firework ID, used in FireworkEntity.java
+    public static final int FIREWORK_ROCKET = 778;
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java:53]: Seems like you don't use this anymore.

**Pull request merged**